source/module_hamilt_lcao/hamilt_lcaodft/FORCE_STRESS.h

```diff
@@ -34,6 +37,10 @@ class Force_Stress_LCAO
                         ModuleBase::matrix& fcs,
                         ModuleBase::matrix& scs,
                         const Structure_Factor& sf,
+#ifdef __EXX
```

Comment:
do not use #ifdef #endif in the middle of a set of input parameters, move them to the end